presto-docs/src/main/sphinx/release/release-0.229.rst

```diff
@@ -0,0 +1,34 @@
+=============
+Release 0.229
+=============
+
+General Changes
+_______________
+* Fix an issue that would cause query failure when calling :func:`geometry_to_bing_tile` on certain degenerate geometries.
```

Comment:
@jagill Is this updated release note accurate? Does the "fail" you mentioned means "query failure"?